crates/solvers/src/domain/order.rs

```diff
@@ -98,11 +98,13 @@ impl<'a> NonLiquidity<'a> {
 #[derive(Clone, Copy, Debug)]
 pub struct Reward(pub f64);
 
+/// Interaction that can be executed as part of an order's pre- or
+/// post-interactions.
 #[derive(Debug, Clone)]
-pub struct CustomInteraction {
+pub struct OrderInteraction {
     pub target: Address,
     pub value: eth::Ether,
-    pub calldata: Vec<u8>,
+    pub call_data: Vec<u8>,
```

Comment:
such a naming nit I almost didn't make this comment:

I've seen this called "Calldata", "Call Data" or just "Data". Personally, I would go for "Calldata" in a single word as it is more distinctive. The only reason we used `callData` in the smart contracts was that `calldata` was a reserved keyword.

Really don't feel strongly at all though, so feel free to hit "resolve thread" and forget I mentioned this.